#1 Group users by lastname..sql

```diff
@@ -0,0 +1,3 @@
+SELECT DISTINCT lastname

```

Comment:
Unfortunately, it is entirely wrong :D `DISTINCT` removes duplicates but your task is to display only "duplicated" last names. Read the issue description again and take a look at [GROUP BY](https://www.w3schools.com/sql/sql_groupby.asp) statement.